#19 get-contexts, current-context, and get-context by name rather default fixed

SiddiqueAhmad opened the pull request:
- get-contexts command added
- current-context command added
- get-context command message changed
- get-context command removed default

gunnarmorling commented on src/main/java/dev/morling/kccli/command/CurrentContextCommand.java:
```diff
@@ -0,0 +1,35 @@
+    public void run() {
+
+        String clusterUri = context.getCurrentContext().getCluster().toASCIIString();
+        System.out.println("Current context '" + context.getCurrentContextName() + "' is set to " + clusterUri);
```
Let's display all the info available from the context object, i.e. also bootstrap server, offset topic, etc. Ideally formatted using the tuple list approach, as for other outputs.

SiddiqueAhmad replied:
okay I will do it

SiddiqueAhmad commented:
@gunnarmorling please review as I am going in right direction, I have added one test case too and modified one to test, current context and get context

gunnarmorling commented on src/main/java/dev/morling/kccli/command/GetContextsCommand.java:
```diff
@@ -0,0 +1,40 @@
+        Map<String, Context> contexts = configContext.getContexts();
+        contexts.forEach((name, context) -> {
+            String clusterUri = context.getCluster().toASCIIString();
+            System.out.println(name + " is set to " + clusterUri);
```
Let's print this in table form here, with `NAME` and `URI`  as the headers.

SiddiqueAhmad replied:
sure, will do that

gunnarmorling commented on src/main/java/dev/morling/kccli/command/GetContextCommand.java:
```diff
@@ -18,17 +18,22 @@
-        String clusterUri = context.getContext().getCluster().toASCIIString();
-        System.out.println("Current context is set to " + clusterUri);
+
+        String clusterUri = context.getContext(contextName).getCluster().toASCIIString();
```
So this changes semantics of this command a bit, as it always has shown the currently active context. Can we keep it that way, when invoking `kcctl config get-context` without any option? And then, when calling `kcctl config get-context --name <NAME>`, show the specified one? I.e. make the name an optional parameter.

SiddiqueAhmad replied:
sure, that makes sense, I even did but could not get how to get the optional params, but your suggestion will help now to do it

gunnarmorling reviewed (commented):
LGTM overall; some minor suggestions inline. Thx a lot, @SiddiqueAhmad!

gunnarmorling commented:
@SiddiqueAhmad, you haven't pushed any update since the last round of feedback above, right?

SiddiqueAhmad commented:
> @SiddiqueAhmad, you haven't pushed any update since the last round of feedback above, right?

yes I am so sorry couldn't manage it yet, weekend was with debezium server

gunnarmorling commented:
No worries, just was curious. Didn't know you're looking into Debezium
Server too :)

>

SiddiqueAhmad commented:
sure, can't stop awesomeness to happen :)
